usage.md

```diff
@@ -0,0 +1,135 @@
+# tcurl(1) -- curl for tchannel
+
+## SYNOPSIS
+
+`tcurl` [--help] [-H] [-p] [-t]
```

Comment:
This seems more like the full help other than synopsis. I guess the idea is to have a very short synopsis and then a full help to follow.